clients/src/main/java/org/apache/kafka/common/utils/Utils.java

```diff
@@ -524,17 +520,7 @@ public static ByteBuffer ensureCapacity(ByteBuffer existingBuffer, int newLength
      * @param separator the separator
      */
     public static <T> CharSequence mkString(Collection<T> coll, String separator) {
```

Comment:
I think we should remove this method since it's no different than `join` and is only used in tests.